scripts/uberenv/uberenv.py

```diff
@@ -380,7 +383,7 @@ def clone_repo(self):
             spack_url = self.project_opts.get("spack_url", "https://github.com/spack/spack.git")
             spack_branch = self.project_opts.get("spack_branch", "develop")
 
-            clone_cmd =  "git {0} clone --single-branch --depth=1 -b {1} {2}".format(clone_opts, spack_branch,spack_url)
+            clone_cmd =  "git {0} clone -b {1} {2}".format(clone_opts, spack_branch,spack_url)
```

Comment:
Won't this change result in bringing over a larger image?  What is the purpose of removing `--single-branch --depth=1`?

Reply:
That may be a mistake of mine! Thanks.